packages/hub/services/in-app-purchases.ts

```diff
@@ -1,8 +1,56 @@
+import config from 'config';
+import fetch from 'node-fetch';
+
+interface InAppPurchaseValidationResult {
+  valid: boolean;
+  response: any;
+}
+
 export default class InAppPurchases {
-  async validate(provider: string, receipt: any) {
-    console.error('In-app purchase validation is not implemented, arguments are', provider, receipt);
+  async validate(provider: string, receipt: any): Promise<InAppPurchaseValidationResult> {
```

Comment:
```suggestion
  async validate(provider: 'google' | 'apple', receipt: string): Promise<InAppPurchaseValidationResult> {
```

If `string` is the correct type for both receipts, then we could do this, is it really `string`?

Reply:
it’s true, this is something I didn’t appreciate initially, thanks